src/controllers/dynakube/istio/reconciler.go

```diff
@@ -77,19 +76,21 @@ func (reconciler *Reconciler) Reconcile(instance *dynatracev1beta1.DynaKube, com
 
 	upd, err := reconciler.reconcileIstioConfigurations(instance, []dtclient.CommunicationHost{apiHost}, "api-url")
 	if err != nil {
-		return false, errors.WithMessage(err, "istio: error reconciling config for Dynatrace API URL")
+		return false, errors.WithMessage(err, "error reconciling config for Dynatrace API URL")
 	} else if upd {
 		return true, nil
 	}
 
 	// Fetch endpoints via Dynatrace client
 	upd, err = reconciler.reconcileIstioConfigurations(instance, communicationHosts, "communication-endpoint")
 	if err != nil {
-		return false, errors.WithMessage(err, "istio: error reconciling config for Dynatrace communication endpoints:")
+		return false, errors.WithMessage(err, "error reconciling config for Dynatrace communication endpoints:")
 	} else if upd {
 		return true, nil
 	}
 
+	log.Info("reconciled")
```

Comment:
is this necessary? 
if we get logs after this without error then it reconciled, also there are multiple none-error cases when we return before even reaching this